plugins/by-name/nvim-ufo/default.nix

```diff
@@ -130,4 +130,15 @@ lib.nixvim.plugins.mkNeovimPlugin {
         end
       '';
   };
+
+  extraConfig = cfg: {
+    plugins.lsp.capabilities = # lua
```

Comment:
I agree this should be gated behind an option. But I'm fine for that option to be `default=true`.

Maybe `setupCapabilities`, `lspCapabilities`, `setupLspCapabilities` or something?

Reply:
I've added the option `settings.setupLspCapabilities`